dnn_test.go

```diff
@@ -283,6 +283,81 @@ func TestTensorflowMemory(t *testing.T) {
 	checkTensorflowNet(t, net)
 }
 
+func TestOnnxMemory(t *testing.T) {
+	path := os.Getenv("GOCV_ONNX_TEST_FILES")
+	if path == "" {
+		t.Skip("Unable to locate ONNX model file for tests")
+	}
+
+	b, err := ioutil.ReadFile(path + "/googlenet-9.onnx")
+	if err != nil {
+		t.Errorf("Failed to load ONNX from file: %v", err)
+	}
+
+	net, err := ReadNetFromONNXBytes(b)
+	if err != nil {
+		t.Errorf("Failed to load Tensorflow model from bytes: %v", err)
+	}
+	if net.Empty() {
+		t.Errorf("Unable to load Tensorflow model")
+	}
+	defer net.Close()
```

Comment:
may as well remove the defer and simply close the net after checkONNXNet

Reply:
it's safe this way IMO